recipes/trinity/build.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/bash
 set -x -e
 
+patch -p1 < ${RECIPE_DIR}/makefile.patch
```

Comment:
Why do you need to run this manually? Specifying the patch as part of meta.yaml will apply it and is more obvious for the reader of the yml file imho.

Reply:
Good catch and I agree that specifying the patch in meta.yml will be easier to understand. I will push a follow-up commit with updates shortly.